src/components/atom_card_frame_surfing_botton/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, useCallback } from 'react'
+import StyledSurfingIcon from '@/atoms/StyledSurfingIcon'
+import { useSemesters } from '@/hooks/semesters/use-semesters.hook'
+import { useWords } from '@/hooks/words/use-words.hook'
+
+const WordCardsSurfingButton: FC = () => {
+  const [, getWords] = useWords()
+  const getSemesters = useSemesters()
+
+  const onClickSurfing = useCallback(async () => {
+    const semesters = await getSemesters()
+    if (!semesters.latestSemesterCode) return
+
+    await getWords({ semester: semesters.latestSemesterCode })
+  }, [getWords, getSemesters])
+
+  return <StyledSurfingIcon onClick={onClickSurfing} runOnClickOnce />
+}
+
+export default WordCardsSurfingButton
```

Comment:
```suggestion
export default WordCardsSurfingButton

```

コードの最後はindentを入れなければいけない

console開いて、(VS code で command + shift + c )すると、白画面のConsoleでるけど、あそこで
`yarn latest`って書いてエンター

じゃ色々でるけど、でた後にVSコード戻ると、Changesに色々でるから、
あそこでmessge `yarn: latest`で書いてcommitつくってください